.gitignore

```diff
@@ -11,11 +11,14 @@ config.log
 
 # Built by sconsUtils
 version.py
-bin/
 
 # Pytest
 tests/.tests
 pytest_session.txt
 .cache/
 .pytest_cache
 .coverage
+ospl-error.log
+ospl-info.log
```

Comment:
Please consider adding `.vscode` for those of us who use that editor.

Also please consider `*.log` instead of these two entries. (Assuming you agree that it is unlikely you will ever want to commit a log file to github.)